File: BLL/ModelsView/StoreTransDep_VM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BLL.ModelsView
{
    public class StoreTransDep_VM
    {
        public decimal? Ptransrowid { get; set; }
        public decimal Ctrnsrowid { get; set; }
        public decimal? Commited { get; set; }
        public decimal? Depslot { get; set; }
        public decimal? Depdetailsid { get; set; }
        public decimal StoreTrnsDepId { get; set; }
        public decimal? Groupid { get; set; }
        public decimal BranchId { get; set; }
        public decimal FromStoreAllcodesId { get; set; }
        public decimal ToStoreAllcodesId { get; set; }
        public string Rem { get; set; }
        public DateTime TrnsDate { get; set; }
        public decimal TrnsCode { get; set; }

        public string BranchName { get; set; }
        public string From_TypeName { get; set; }
        public string TO_TypeName { get; set; }
        public string From_StoreAllcodesName { get; set; }
        public string To_StoreAllcodesName { get; set; }
    }
}

File: Angular_UIL/ClientApp/src/app/models/Transactions/StoreTransaction/SaveStoreTransaction/storeTransDep_VM.ts
export class storeTransDep_VM {
  constructor(
    public Ptransrowid?: number,
    public Ctrnsrowid?: number,
    public Groupid?: number,
    public TrnsCode?: number,
    public TrnsNo?:number,
    public ToStoreAllcodesId?: number,
    public FromStoreAllcodesId?: number,
    public BranchId?: number,
    public StoreTrnsDepId?: number,
    public Depdetailsid?: number,
    public Depslot?: number,
    public Commited?: number,
    public TrnsDate?:Date,
    public Rem?: string,
    public BranchName?: string,
    public From_TypeName?: string,
    public TO_TypeName?: string,
    public From_StoreAllcodesName?: string,
    public To_StoreAllcodesName?: string,
  ) { }
}
